docs/consume-packages/Finding-and-Choosing-Packages.md

```diff
@@ -108,13 +108,13 @@ The Visual Studio Package Manager Console, NuGet CLI, and dotnet CLI tools don't
 <a name="native-cpp-packages"></a>
 ### Native C++ packages
 
-Visual Studio C++ projects can use native C++ NuGet packages. Installing these packages enables the **Manage NuGet Packages** context-menu command, exposes a `native` target framework, and provides MSBuild integration.
+Visual Studio C++ projects can use native C++ NuGet packages. Installing these packages enable the **Manage NuGet Packages** context-menu command, exposes a `native` target framework, and provides MSBuild integration.
 
 To find native packages on nuget.org/packages, search by using `tag:native`. Such packages typically provide *.targets* and *.props* files, which NuGet imports automatically when adding the packages.
 
 ## Evaluate packages
 
-The best way to evaluate a package's usefulness is to try it out. You take a dependency on a package when you use it, so you must make sure it's robust and reliable. However, installing a package and directly testing it is time-consuming. You can learn a lot about a package's quality by using the information on the package's page at nuget.org/packages.
+The best way to evaluate a package's usefulness is to try it out. You take a dependency on a package when you use it, so you must make sure it is robust and reliable. However, installing a package and directly testing it, is time-consuming. You can learn a lot about a package's quality by using the information on the package's page at nuget.org/packages.
```

Comment:
I disagree with "it is" instead of "it's". Although, I don't know if Microsoft's docs guidelines have changed, because it's really hard to find now. In the past, the guidance for docs was to use a "conversational" style, as it's generally easier to understand, and feels more "friendly", than formal language.

Also, I don't know about the comma. In fact, when I copy-paste the sentence into Word, its grammar checker suggests removing the comma. I found the BBC have a page explaining some usages of commas: https://www.bbc.co.uk/bitesize/topics/zvwwxnb/articles/zc773k7. I don't see how the proposed comma fits any of the rules/recommendations from that page. [Grammarly have a long page explaining commas](https://www.grammarly.com/blog/comma/), but honestly, it's so long with so many different sections, it's too difficult/time consuming to figure out how their suggestions work in relation to this sentence.

Reply:
Yes I surely do agree with you at this one but there is this thing that , *It's* is the [possessive]  form of *it* , used to indicate possession, ownership, belonging, etc. And in this case we surely aren't talking about the possession of  `packages` , so in my opinion it would be better to go with *it is* . 
[Use this as reference](https://www.dictionary.com/e/its-vs-its/)

Reply:
`it's` can be used in possessive form, but it's also a contraction of `it is`. As linked in the other comment [Microsoft style guide for docs explicitly says "write like you speak"](https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice#write-like-you-speak). In fact, the very next guideline [project friendiness explicitly says "use contractions"](https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice#project-friendliness)

Reply:
Sure thing 👍🏿 
Besides the words you've mentioned other one is correct right? `for` one? Should I revert back these words to their original forms and then can you merge the PR?

Reply:
The "enables" fix is definately correct. I think the "for" change can go either way, but I don't care. It's just the two lines (three changes) that I commented on that I didn't agree with.  Thanks for making the changes!